src/installation/live-images/guide.md

```diff
@@ -42,8 +42,8 @@ To install packages provided on the install image, select `Local`. Otherwise,
 you may select `Network` to download the latest packages from the Void
 repository.
 
-> Note: if you are installing a desktop environment from a ''flavor'' image, you
-> MUST choose `Local` for the source!
+Note: if you are installing a desktop environment from a ''flavor'' image, you
+MUST choose `Local` for the source!
```

Comment:
Vote for making this a warning.

Reply:
that has my support. I'm also going to lowercase must and change the double-single quotes into single-double quotes (unless there are any objections)

Reply:
i've changed this in #355 (not yet merged). i think this PR might also be affected by several recent changes - you should rebase and check.

Reply:
This branch is up to date! I rebased earlier tonight. There was only a couple conflicts I had to fix.